pub/lifecycle.go

```diff
@@ -27,13 +27,13 @@ type lifecycleErrorResponse struct {
 }
 
 type PublicEndpointRequestDto struct {
-	FatmanName    string `json:"fatman_name"`
-	FatmanVersion string `json:"fatman_version"`
+	JobName    string `json:"job.name"`
+	JobVersion string `json:"job.version"`
```

Comment:
Why are these dots here?
```suggestion
	JobName    string `json:"job_name"`
	JobVersion string `json:"job_version"`
```

Reply:
Because I made the mistake of not escaping a dot I presume. `.` is a wildcard single character. Nice catch